test/oqs_test_endecode.c

```diff
@@ -175,6 +175,11 @@ static int test_oqs_encdec(const char *alg_name) {
         if (pkey == NULL)
             goto end;
 
+        if (!OBJ_sn2nid(alg_name)) {
+            printf("No OID registered for %s", alg_name);
+            ok = 1;
```

Comment:
May I suggest that we mark these as "skipped" instead of "passed"? We might be grateful for it later if we end up revisiting this code---we'll be able to tell from output alone if the tests are actually running for a given algorithm.

I'll add a suggestion to the test code as well to handle the extra return value.
```suggestion
            ok = -1;
```

Reply:
OK, it seems that I can't add a suggestion directly on unmodified lines of code, but we could replace lines 221--231 with

```c
switch(test_oqs_encdec(algs->algorithm_names)) {
    case 1:
        fprintf(stderr,
                cGREEN "  Encoding/Decoding test succeeded: %s" cNORM "\n",
                algs->algorithm_names);
        break;
    case -1:
        fprintf(stderr,
                cBLUE "  Encoding/Decoding test skipped: %s" cNORM "\n",
                algs->algorithm_names);
        break;
    default:
        fprintf(stderr,
                cRED "  Encoding/Decoding test failed: %s" cNORM "\n",
                algs->algorithm_names);
        ERR_print_errors_fp(stderr);
        errcnt++;
        break;
}
```

Reply:
If you think this is a good idea, I can commit it directly to the branch to save you the trouble of copy-paste-format.

Reply:
@baentsch are you OK with my pushing the above code snippet to print "skipped" instead of "passed" on a -1 return code?